data/i18n/en/en.toml

```diff
@@ -355,6 +355,64 @@ other = "Before you begin"
 [previous_patches]
 other = "Patch Releases:"
 
+[release_binary_alternate_links]
```

Comment:
Consider adding a comment to localizers that this is used when JavaScript isn't available.